easyunfold/cli.py

```diff
@@ -350,37 +387,117 @@ def print_data(entries, tag='me'):
+                zero_line, dos_elements, dos_orbitals, dos_atoms, legend_cutoff, gaussian, no_total, total_only, scale, procar, atoms,
+                atoms_idx, orbitals, title, width, height, dpi, mpl_style_file):
     """
     Plot the spectral function
 
     This command uses the stored unfolding data to plot the effective bands structure (EBS) using the spectral function.
     """
-    _unfold_plot(ctx, gamma, npoints, sigma, eref, out_file, show, emin, emax, cmap, ncl, no_symm_average, vscale, procar, atoms_idx,
-                 orbitals, title, width, height, dpi)
+    if mpl_style_file:
+        click.echo(f'Using custom plotting style from {mpl_style_file}')
+        import matplotlib.style
+        matplotlib.style.use(mpl_style_file)
+
+    _unfold_plot(ctx, gamma, npoints, sigma, eref, out_file, show, emin, emax, cmap, ncl, no_symm_average, colour_norm, dos, dos_label,
+                 zero_line, dos_elements, dos_orbitals, dos_atoms, legend_cutoff, gaussian, no_total, total_only, scale, procar, atoms,
+                 atoms_idx, orbitals, title, width, height, dpi)
 
 
 @unfold.command('plot-projections')
 @click.pass_context
 @add_plot_options
 @click.option('--combined/--no-combined', is_flag=True, default=False, help='Plot all projections in a combined graph.')
-@click.option('--intensity', default=1.0, help='Color intensity for combined plot', type=float, show_default=True)
-@click.option('--colors', help='Colors to be used for combined plot, comma separated.', default='r,g,b,purple', show_default=True)
-def unfold_plot_projections(ctx, gamma, npoints, sigma, eref, out_file, show, emin, emax, cmap, ncl, no_symm_average, vscale, procar,
-                            atoms_idx, orbitals, title, combined, intensity, colors, width, height, dpi):
+@click.option('--colours', help='Colours to be used for combined plot, comma separated.', default='r,g,b,purple', show_default=True)
+def unfold_plot_projections(ctx, gamma, npoints, sigma, eref, out_file, show, emin, emax, cmap, ncl, no_symm_average, colour_norm, dos,
+                            dos_label, zero_line, dos_elements, dos_orbitals, dos_atoms, legend_cutoff, gaussian, no_total, total_only,
+                            scale, procar, atoms, atoms_idx, orbitals, title, combined, colours, width, height, dpi, mpl_style_file):
     """
     Plot the effective band structure with atomic projections.
     """
     from easyunfold.unfold import UnfoldKSet
     from easyunfold.plotting import UnfoldPlotter
     import matplotlib.pyplot as plt
 
+    if mpl_style_file:
+        click.echo(f'Using custom plotting style from {mpl_style_file}')
+        import matplotlib.style
+        matplotlib.style.use(mpl_style_file)
+
     unfoldset: UnfoldKSet = ctx.obj['obj']
     click.echo(f'Loading projections from: {procar}')
 
     plotter = UnfoldPlotter(unfoldset)
+    if dos:
```

Comment:
Yep good point, should've done this from the start. Was just from the way I started coding it (for the projected plot first, then realising it could go in the other and thinking the handling would differ significantly etc etc). Done now! 😃